torch/_subclasses/fake_tensor.py

```diff
@@ -1161,6 +1161,10 @@ def dispatch(self, func, types, args=(), kwargs=None):
             func, converter, args, kwargs
         )
 
+        # Recompute flat_arg_fake_tensors here again in case some of the inputs
+        # were real tensors and fakified in validate_and_convert_non_fake_tensors
+        flat_arg_fake_tensors = tree_flatten_only(FakeTensor, (args, kwargs))
+
```

Comment:
This is not free! What is the perf overhead here? Do you need help kicking off a measurement run?

Reply:
Yes, that would be great. Thanks.

Reply:
You could move the creation of this to `validate_and_convert_non_fake_tensors` ? would be faster ?